dlio_benchmark/checkpointing/pytorch_checkpointing.py

```diff
@@ -42,14 +59,16 @@ def __init__(self):
         super().__init__("pt")
 
     @dlp.log
-    def get_tensor(self, size):
-        return torch.randint(high=1, size=(size,), dtype=torch.int8)
+    def get_tensor(self, size, datatype="int8"):
```

Comment:
The API is different from what I had. What I mean is size is inclusive of type of data. Based on what you want to do, you can do array_length? or length instead of size.

Reply:
can you fix the size variable to be length?